leaguebot/services/screeps.py

```diff
@@ -7,8 +7,8 @@
 def get_client():
     conn = getattr(g, '_screeps_client', None)
     if conn is None:
-        user = app.config['API_USERNAME']
-        password = app.config['API_PASSWORD']
+        user = app.config['API_USERNAME'] or None
+        password = app.config['API_PASSWORD'] or None
```

Comment:
This isn't strictly related, but I needed to add this in order to allow for testing without an API login set.